engineering_tools/remote_install.bash

```diff
@@ -112,11 +112,10 @@ LOCAL_CARMA_DIR=${PWD}
 echo "Installing to ${HOST} as user: ${USERNAME}..."
 
 # Define paths for files to copy from src
```

Comment:
You can delete this whole file. It's left over from carma2 and was replaced by the carma script.